package com.tmd.common.entity.dto;


import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class MailDTO {
    private String stampType;
    private String stampContent;
    private String senderNickname;
    private String recipientEmail;
    private String content;
}
